### PR TITLE
An 1212/gold events

### PR DESCRIPTION
**Pull request by forgxyz**

### CODE REVIEW

File: models/gold/gold__events.sql [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            attribute_value_adj :: variant
+        ) AS event_data
+    FROM
+        silver_event_attributes
+    GROUP BY
+        1
+),
+location_object AS (
+    SELECT
+        event_id,
```
Comment (jkallogjeri) [inline models/gold/gold__events.sql:45]: Why does event_id include the tx_id in the column? Is it necessary for what the column is trying to accomplish? 

Can "2f572a7fc5af54249f819a974fc69c6da21a6691d82560dc659cca1fdf68747c-10" not just be "10"

Comment (jkallogjeri) [reply models/gold/gold__events.sql:45]: Actually I see there is already a "event_index" field. We can just remove this column all together.

File: models/core/core__fact_events.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      - name: event_type
+        description: "{{ doc('event_type') }}"
+        tests:
+          - not_null
+          - dbt_expectations.expect_column_values_to_be_in_type_list:
+              column_type_list:
+                - STRING
+                - VARCHAR
+      
+      - name: event_attributes
```
Comment (jkallogjeri) [inline models/core/core__fact_events.yml:80]: this needs to be event_data not even_attributes

File: models/core/core__fact_events.yml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        description: "{{ doc('event_attributes') }}"
+        tests:
+          - not_null
+          - dbt_expectations.expect_column_values_to_be_in_type_list:
+              column_type_list:
+                - ARRAY
```
Comment (jkallogjeri) [inline models/core/core__fact_events.yml:86]: I don't think it's an Array? it's an object no since it's with {} instead of []

Comment (jkallogjeri) [reply models/core/core__fact_events.yml:86]: test is failing cus of it

File: models/gold/gold__events.yml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+      - name: event_type
+        description: "{{ doc('event_type') }}"
+        tests:
+          - not_null
+          - dbt_expectations.expect_column_values_to_be_in_type_list:
+              column_type_list:
+                - STRING
+                - VARCHAR
+      
+      - name: event_attributes
```
Comment (jkallogjeri) [inline models/gold/gold__events.yml:80]: Same comments as above here

File: models/gold/gold__events.sql (hunk 1 above)
Comment (forgxyz) [reply models/gold/gold__events.sql:45]: event_id is used in silver & gold as a unique key for joining after transforms. I can drop from the final gold view in core, but will leave it here in the upstream CTEs

File: models/core/core__fact_events.yml (hunk 3 above)
Comment (forgxyz) [reply models/core/core__fact_events.yml:86]: ty, forgot to change the name and type when I updated the structure

**Review by jkallogjeri — APPROVED** [review]

**Pull request merged**